vite.config.js

```diff
@@ -0,0 +1,10 @@
+import {defineConfig} from "vite";
+import react from "@vitejs/plugin-react";
+
+export default defineConfig({
+    plugins: [react()],
+    server: {
+        port: 3000,
+        open: true
+    }
+})
```

Comment:
can use plugins like vite-plugin-compression to compress your build files, which can help to reduce the size of your bundle and improve load times.
```suggestion
import {defineConfig} from "vite";
import react from "@vitejs/plugin-react";
import compress from 'vite-plugin-compression'

export default defineConfig({
    plugins: [
        react(),
        compress()
    ],
    server: {
        port: 3000,
        open: true
    }
})
```